tools/importers/SynchronizeTypographyTokens.ts

```diff
@@ -0,0 +1,191 @@
+const axios = require('axios');
+const tokensFilesConfig = require('./configs/SynchronizeTypographyTokensConfig.json');
+import { arrayToMixinFile, arrayToFile, jsonToFile } from './helpers/fileWriter';
+import { recursiveTokensReader } from './helpers/tokensReader';
+import {
+	typographyPrefix,
+	tokensKey,
+	excludedKeys,
+	importVariables,
+	jsonAttributeToCssProperty,
+	resultCssAdditionalLines,
+	resultScssAdditionalLines,
+	tokensTypographyInputAttributes,
+} from './helpers/typographyVariables';
+import {
+	cssFileFirstLine,
+	mixinNameLine,
+	tokenAsCssPropertyLine,
+	closeBracketLine,
+} from './helpers/modifiers';
+import * as dotenv from 'dotenv';
+import {
+	Dict,
+	ITokenJsonObject,
+	IResultJsonObject,
+	ITypographyToken,
+	TypographyConfigFileBin,
+	TypographyBinFiles,
+	JsonAttribute,
+	JsonAttributeValues,
+} from './helpers/structures';
+
+dotenv.config();
+
+const ImportTypographyVariables = (binFilesConfig: TypographyBinFiles, jsonTypography: Object) => {
+	let resultCss: Array<string> = [];
+	let resultScss: Array<string> = [];
+	let resultJsonCss: Dict<Array<IResultJsonObject>> = {};
+
+	resultCss.push(cssFileFirstLine(false, undefined));
```

Comment:
maybe we should have a separate method for theme and for `:root` css block start?

Reply:
Sure, splitted.